diff --git a/node_modules/@types/pino/index.d.ts b/node_modules/@types/pino/index.d.ts
index 155297e..255c7dc 100755
--- a/node_modules/@types/pino/index.d.ts
+++ b/node_modules/@types/pino/index.d.ts
@@ -20,7 +20,7 @@
 import stream = require("stream");
 import http = require("http");
 import { EventEmitter } from "events";
-import SonicBoom = require("sonic-boom");
+type SonicBoom = any;
 import * as pinoStdSerializers from "pino-std-serializers";
 import * as PinoPretty from "pino-pretty";
 
